frontend/src/components/filters/FiltersInfos.js

```diff
@@ -31,6 +31,10 @@ const FiltersInfos = ({
         if (value !== "")
           return value
         throw new Error('MIN and MAX values not defined for Filter range')
+      case FILTER_TYPE.METADATA:
+        return valuesArray.reduce((metadataString, metadata) => {
+          return metadataString + metadata.name + ": " + (metadata.value ? metadata.value : "any") + ', '
```

Comment:
Maybe use "*" instead of "any" for the case where the actual content of the field would be any. * is a bit less likely. You could also use an icon to make it clear it is not looking for an user input.